src/api/syscall.c

```diff
@@ -222,10 +224,11 @@ exception_t handleUnknownSyscall(word_t w)
 
 exception_t handleUserLevelFault(word_t w_a, word_t w_b)
 {
-    MCS_DO_IF_BUDGET({
```

Comment:
Again, this is not a notable improvement. Please don't merge.

Reply:
I think replacing the macro by a clearly visible if-statement with a condition is is an improvement in terms of making it easier to understand what the code actually does. I've spend some time understanding this, especially since this macro wraps big code blocks. Furthermore, it seem this macro prevents some code analyzers to see the code block, because it is just removed - at least this would explain why the style checker now complains. Now the style check is arguably not the most important thing, but there might be other tools also.